src/extension/wps/wps-core/src/test/java/org/geoserver/wps/gs/ImportProcessTest.java

```diff
@@ -31,6 +35,10 @@
     @After 
     public void removeNewLayers() {
         removeLayer(SystemTestData.CITE_PREFIX, "Buildings2");
+        removeLayer(SystemTestData.CITE_PREFIX, "Buildings4");
+        removeLayer(SystemTestData.CITE_PREFIX, "Buildings5");
+        removeStore(SystemTestData.CITE_PREFIX, SystemTestData.CITE_PREFIX + "data");
+        removeStore(SystemTestData.CITE_PREFIX, SystemTestData.CITE_PREFIX + "raster");
```

Comment:
Right, I saw the other tests were deleting the layer they used, so I followed that example and deleted what the tests created.